web/src/components/ui/Button.module.css

```diff
@@ -1,44 +1,11 @@
 .button {
   display: block;
   border-radius: 6px;
```

Comment:
I think we should stick with the UCL style and not have any rounding on anything